openjdk_regression/playlist.xml

```diff
@@ -146,6 +146,29 @@
 		</impls>
 		<disabled>https://github.com/AdoptOpenJDK/openjdk-tests/issues/228</disabled>
 	</test>
+	<test>
+		<testCaseName>runtime_nestmate</testCaseName>
+		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(TEST_RESROOT)$(D)jtreg$(D)lib$(D)jtreg.jar$(Q) \
+	$(JTREG_BASIC_OPTIONS) -vmoptions:$(Q)-Xmx512m $(JVM_OPTIONS)$(Q) \
+	-w $(Q)$(TEST_RESROOT)$(D)work$(Q) \
+	-r $(Q)$(TEST_RESROOT)$(D)report$(Q) \
+	-jdk:$(Q)$(JDK_HOME)$(Q) \
+	-exclude:$(Q)$(TEST_RESROOT)$(D)ProblemList_$(JVM_VERSION).txt$(Q) \
+	$(Q)$(JTREG_HOTSPOT_TEST_DIR)$(D)runtime$(D)Nestmates$(Q); \
+	$(TEST_STATUS)</command>
+		<subsets>
+			<subset>SE110</subset>
+		</subsets>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>openjdk</group>
+		</groups>
+		<impls>
+			<impl>openj9</impl>
```

Comment:
Nestmates is actually a subset of hotspot_runtime. For hotspot I'm thinking maybe it's better to enable hotspot_runtime or even a larger subset than hotspot_runtime? Limiting it to openj9 will not let it run twice. 

Since JDK10andup the hotspot groups changed a lot we will need to revisit and update to run newly added or re-structured tests.

Reply:
yes, agreed.  thanks for the more detailed info (didn't realize these tests were a subset of the hotspot_runtime target).  

Once you fix the merge conflicts we can merge.